src/components/ResizableBar.vue

```diff
@@ -0,0 +1,376 @@
+<template>
+  <div
+    class="resizable-bar"
+    :style="{
+      bottom: sidebarBottom + 'px',
+      height: `${Math.min(sidebarHeight, maxHeight)}px`,
+    }"
+  >
+    <div class="resizable-bar__header" @mousedown="startResizing">
+      <hr :class="{ 'green-color': isResizing }" />
+      <button @click="resizeBar">
+        <img src="../assets/images/icons/expand_circle_up.png" alt="Seta" />
+      </button>
+      <hr :class="{ 'green-color': isResizing }" />
+    </div>
+    <div class="resizable-bar__content">
+      <div v-if="visibleInsights" class="insights">
+        <div>
+          <p>InsightsGPT:</p>
+          <h2>Aqui estão algumas sugestões para você:</h2>
+        </div>
+        <div class="insights-content">
+          <div class="insights-list">
+            <ul>
+              <li>Recentes</li>
+              <li>Segmentos</li>
+              <li>Varejo</li>
+              <li>Weni plataforma</li>
+              <li class="selected">Atendimento humano</li>
+            </ul>
+          </div>
+          <div class="insights-cards">
```

Comment:
There is a lot of repeated code in this section, would it be viable to transform the cards into reusable components?

Reply:
Of course, it would be great to avoid repeated code.